code/__DEFINES/MC.dm

```diff
@@ -20,8 +20,9 @@
 
 //SubSystem flags (Please design any new flags so that the default is off, to make adding flags to subsystems easier)
 
+//deprecated, can be replaced
 //subsystem should fire during pre-game lobby.
-#define SS_FIRE_IN_LOBBY 1
+//#define SS_FIRE_IN_LOBBY 1
```

Comment:
I mean, I think to show that the flag is open, but he could just re-shift them around, the point of using defines is you can do that without side effects